velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -129,5 +128,38 @@ TEST_F(StringAggregatesTest, nonFixedWidthAggregate) {
   testWindowFunction(input, "max(c2)", kOverClauses);
 }
 
+class AggregateEmptyFramesTest : public WindowTestBase {};
+
+// Test for aggregates that return NULL as the default value for empty frames
+// against DuckDb.
+TEST_F(AggregateEmptyFramesTest, nullEmptyResult) {
+  auto input = {makeSinglePartitionVector(50), makeSinglePartitionVector(40)};
+  auto aggregateFunctions = kAggregateFunctions;
+  aggregateFunctions.erase(
+      std::remove(
+          aggregateFunctions.begin(), aggregateFunctions.end(), "count(c2)"),
+      aggregateFunctions.end());
+  for (const auto& windowFunction : aggregateFunctions) {
```

Comment:
nit: it is confusing to mix window and aggregate functions here; maybe

```
for (const auto& function : aggregateFunctions)
```

Reply:
Fixed.